### ISSUE TITLE
Polls don't work

### ISSUE THREAD

**Issue by winks** [toddsundsted/ktistec#49]
I'm not exactly sure *how* polls should work, but they don't.

If I search for https://mstdn.social/@rysiek/109365822245628675 (as long as it's open - or any other "poll" post), I only see the text, not the poll.

In case this is really some Mastodon-only feature, it might make sense to display something, if the meta data is there. This used to be weird with some Twitter clients in the past as well, silently ignoring the poll itself is a bad user experience.

**Comment by toddsundsted** [comment]
i'm not sure what the status is now, but when i last looked polls were something mastodon invented but other servers did not support. that's not uncommon—they all have their custom features. supporting every feature of every server isn't really possible in practice.

i suspect the issue with presentation is ktistec sanitizing out the html tags that present the status of the poll. that's probably not too hard to fix. full support, including participation, wouldn't be trivial.

i'll keep the suggestion open, but being honest, it's not likely to get my attention soon, unfortunately, unless polls have garnered support from other servers and are more broadly implemented. (i don't want to chase features that only one server supports, no matter how popular that one server is.)

**Comment by toddsundsted** [comment]
it looks like ActivityPub includes a Question object type, which underpins mastodon polls, so maybe there is a way forward with this.

**Comment by toddsundsted** [comment]
and it looks like i even implemented the Question object type, though it's just a placeholder.

**Comment by winks** [comment]
Yeah, my point is more like "you should notice that there is a poll to click through" is important.
